localization/ekf_localizer/include/ekf_localizer/state_transition.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018-2019 Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef EKF_LOCALIZER__STATE_TRANSITION_HPP_
+#define EKF_LOCALIZER__STATE_TRANSITION_HPP_
+
+#include "ekf_localizer/matrix_types.hpp"
+
+double NormalizeYaw(const double & yaw);
```

Comment:
I think it is better to follow lowerCamerCase. The same goes for other fuctions defined here, too.

```suggestion
double normalizeYaw(const double & yaw);
```

Reply:
The ros2 naming convention says CamelCase. 
It also says that `for existing projects, prefer the existing style` but in this occasion we can consider renaming the functions 

https://docs.ros.org/en/foxy/The-ROS2-Project/Contributing/Code-Style-Language-Versions.html#function-and-method-naming

Reply:
I understood the ROS 2 naming conventions, but IMHO it is better to leave all the functions in `lowerCamelCase` since most of the functions in Autoware follows this style. Maybe it is better to ask someone else who is more familiar with this type of issue in Autoware.

Reply:
OK. It would be better to change them all at once if we really do it, so I renamed the functions in b2ab7c3